src/librustc_data_structures/flock.rs

```diff
@@ -16,6 +16,12 @@ cfg_if! {
         use std::os::unix::prelude::*;
         use libc;
 
+        pub const F_RDLCK: libc::c_short = libc::F_RDLCK as libc::c_short;
```

Comment:
Why is it necessary to have these as constants here?

Reply:
Good point. I'll push an updated version in a second.

Reply:
Fixed :).